src/scr_conf.h

```diff
@@ -144,12 +144,12 @@
 /* buffer size to use for MPI send / recv operations */
 #ifndef SCR_MPI_BUF_SIZE
 /* #define SCR_MPI_BUF_SIZE (1*1024*1024) */
-#define SCR_MPI_BUF_SIZE (128*1024)  /* very strange that this lower number beats the upper one, but whatever ... */
+#define SCR_MPI_BUF_SIZE (1024*1024)  /* very strange that this lower number beats the upper one, but whatever ... */
```

Comment:
Thanks @hbchen1984 .  Since this number now matches the number that is commented out above, let's drop both of these comments -- the comment above and the comment at the end of this line.